pkg/proxy/iptables/proxier_test.go

```diff
@@ -480,17 +497,17 @@ COMMIT
 :KUBE-NODEPORTS - [0:0]
 :KUBE-POSTROUTING - [0:0]
 :KUBE-MARK-MASQ - [0:0]
-:KUBE-SVC-XPGD46QRK7WJZT7O - [0:0]
-:KUBE-SEP-SXIVWICOYRO3J4NJ - [0:0]
+:KUBE-SVC-ns1/svc1:p80-tcp - [0:0]
+:KUBE-SEP-ns1/svc1:p80-tcp - [0:0]
 -A KUBE-POSTROUTING -m mark ! --mark 0x4000/0x4000 -j RETURN
 -A KUBE-POSTROUTING -j MARK --xor-mark 0x4000
 -A KUBE-POSTROUTING -m comment --comment "kubernetes service traffic requiring SNAT" -j MASQUERADE
 -A KUBE-MARK-MASQ -j MARK --or-mark 0x4000
--A KUBE-SERVICES -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41 --dport 80 -j KUBE-SVC-XPGD46QRK7WJZT7O
--A KUBE-SVC-XPGD46QRK7WJZT7O -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41 --dport 80 ! -s 10.0.0.0/24 -j KUBE-MARK-MASQ
--A KUBE-SVC-XPGD46QRK7WJZT7O -m comment --comment ns1/svc1:p80 -j KUBE-SEP-SXIVWICOYRO3J4NJ
--A KUBE-SEP-SXIVWICOYRO3J4NJ -m comment --comment ns1/svc1:p80 -s 10.180.0.1 -j KUBE-MARK-MASQ
--A KUBE-SEP-SXIVWICOYRO3J4NJ -m comment --comment ns1/svc1:p80 -m tcp -p tcp -j DNAT --to-destination 10.180.0.1:80
+-A KUBE-SERVICES -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41 --dport 80 -j KUBE-SVC-ns1/svc1:p80-tcp
+-A KUBE-SVC-ns1/svc1:p80-tcp -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41 --dport 80 ! -s 10.0.0.0/24 -j KUBE-MARK-MASQ
+-A KUBE-SVC-ns1/svc1:p80-tcp -m comment --comment ns1/svc1:p80 -j KUBE-SEP-ns1/svc1:p80-tcp
+-A KUBE-SEP-ns1/svc1:p80-tcp -m comment --comment ns1/svc1:p80 -s 10.180.0.1 -j KUBE-MARK-MASQ
+-A KUBE-SEP-ns1/svc1:p80-tcp -m comment --comment ns1/svc1:p80 -m tcp -p tcp -j DNAT --to-destination 10.180.0.1:80
```

Comment:
for consistency with "real" data, the SEP chain names in the parsing tests should include the endpoint IP too. eg, `KUBE-SEP-ns1/svc1:p80-tcp-10.180.0.1:80`.

(actually, some of the chains in the "real" tests are wrong too... I guess you just did search+replace?)

Reply:
I did just s+r.  :)  A little better now.